schema/workflow.json

```diff
@@ -414,12 +414,12 @@
               "description": "Name of the referenced function"
             },
             "arguments": {
-              "type": "object",
+              "$ref": "common.json#/definitions/objectOrMarkup",
               "description": "Function arguments/inputs"
             },
             "selectionSet": {
-              "type": "string",
-              "description": "Only used if function type is 'graphql'. A string containing a valid GraphQL selection set"
+              "$ref": "common.json#/definitions/stringOrMarkup",
+              "description": "Only used if function type is 'graphql'. A string or an expression returning a string formatted as a valid GraphQL selection set"
```

Comment:
" A string or an expression returning a string formatted as a valid GraphQL selection set"
i really have 0 clue what this means as a reader. does it have to be a string just cause its a string value property? 
any object return by expression can be expressed in its string format here imo. but could be wrong.

Reply:
Agree, it is easier to read when keeping "contains" like it was before. So "A string (or an expression returning a string) containing a valid GraphQL selection set" is easier to read.
The stringOrMarkup is to allow the parser to already detect whether it's an expression that should be fed to the interpreter or whether it's a string that can remain what it is.